moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -607,64 +837,91 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
               <CircularProgress />
             ) : data && data["project_list_view"] ? (
               <Card className={classes.root}>
-                <MaterialTable
-                  onChangeColumnHidden={handleColumnChange}
-                  data={data["project_list_view"]}
-                  columns={columns}
-                  title=""
-                  options={{
-                    search: false,
-                    rowStyle: {
-                      fontFamily: typography.fontFamily,
-                      fontSize: "14px",
+                <DataGrid
+                  getRowHeight={() => "auto"}
```

Comment:
if we dont do this, the rows are all the same height. but then we have to add the padding like below